zstash/globus.py

```diff
@@ -157,6 +157,7 @@ def file_exists(name: str) -> bool:
             return True
     return False
 
+gv_push = 0
```

Comment:
Why `gv_push`? A more descriptive name might be better. Maybe `tar_file_count`?

Reply:
True, but it was just a way for me to track things.  We could change it.

I wanted a variable to track "actual transfer submitted" (pushed), as opposed to just submitted to our globus_transfer() function, which may just add it to a pending transfer and return.  I'll make it "gv_tarfiles_pushed".